srml/contracts/src/exec.rs

```diff
@@ -120,6 +121,9 @@ pub trait Ext {
 
 	/// Rent allowance of the contract
 	fn rent_allowance(&self) -> BalanceOf<Self::T>;
+
+	// Returns the current block number.
```

Comment:
```suggestion
	/// Returns the current block number.
```

Reply:
@pepyakin Thank you for suggesting! It fixed.